### PR TITLE
ap-southeast-3 (Jakarta) / ap-northeast-3 (Osaka)

### PR DESCRIPTION
**Pull request by PatMyron**
https://aws.amazon.com/blogs/aws/aws-asia-pacific-osaka-region-now-open-to-all-with-three-azs-more-services/
https://aws.amazon.com/blogs/aws/now-open-aws-asia-pacific-jakarta-region/

---

not sure if this mapping is still used since there was another mapping updated in https://github.com/infracost/infracost/pull/1236, but updating it based on https://github.com/aws-cloudformation/cfn-lint/blob/main/scripts/update_specs_from_pricing.py#L19

---

also guessing isolated regions probably don't have pricing available:
https://github.com/infracost/cloud-pricing-api/blob/001bbc1b7e791e5f7ab47449e4efb734e5eaa1bb/src/scrapers/awsBulk.ts#L24-L25

### CODE REVIEW

**Comment by alikhajeh1** [comment]
Thanks @PatMyron - sorry we dropped the ball on this PR

File: src/scrapers/awsBulk.ts [hunk 1]
```diff
@@ -13,7 +13,6 @@ const chinaIndexUrl = '/offers/v1.0/cn/index.json';
 const splitByRegions = ['AmazonEC2'];
 
 const regionMapping: { [key: string]: string } = {
-  'AWS GovCloud (US)': 'us-gov-west-1',
```
Comment (aliscott) [inline src/scrapers/awsBulk.ts:16]: We should keep this. The mappings are for converting locations from the AWS bulk pricing API data to the actual region codes, and sometimes they come through as "AWS GovCloud (US)".

File: src/scrapers/awsBulk.ts [hunk 2]
```diff
@@ -35,9 +34,10 @@ const regionMapping: { [key: string]: string } = {
   'Asia Pacific (Hong Kong)': 'ap-east-1',
   'Asia Pacific (Tokyo)': 'ap-northeast-1',
   'Asia Pacific (Seoul)': 'ap-northeast-2',
-  'Asia Pacific (Osaka-Local)': 'ap-northeast-3',
```
Comment (aliscott) [inline src/scrapers/awsBulk.ts:38]: As above, we should also keep this one since we've seen the data come through with this location.

**Review by aliscott — COMMENTED** [review]
Thanks @PatMyron. I've suggested a couple of changes. Let me know what you think. We use these mappings to normalize the regions coming through from the AWS bulk pricing API.

For the isolated regions, we do see data come through form them, but I'm not sure how that works with Terraform.

File: src/scrapers/awsBulk.ts (hunk 1 above)
Comment (PatMyron) [reply src/scrapers/awsBulk.ts]: thought AWS had fixed these old region names from surfacing in pricing anymore, but added them both back just in case

**Comment by PatMyron** [comment]
> For the isolated regions, we do see data come through form them, but I'm not sure how that works with Terraform.

Interesting, I believe they're [mostly supported through the AWS Go SDK](https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/#pkg-constants). Terraform also has [`skip_region_validation`](https://registry.terraform.io/providers/hashicorp/aws/latest/docs#skip_region_validation) in case there are any issues on their end

**Review by aliscott — APPROVED** [review]
Thanks @PatMyron!

**Pull request merged**